ci-operator/config/stackrox/stackrox/stackrox-stackrox-master.yaml

```diff
@@ -64,6 +64,27 @@ resources:
       cpu: 2000m
       memory: 4000Mi
 tests:
+- as: push-images
+  steps:
+    test:
```

Comment:
@house-d I was not able to use the `stackrox-push` step here. The image set is going to be different across the three main repos and they have to be injected as `dependencies` and I'm not sure of a good way to do that. Given that there will only be 1 or 2 pushes per repo, it is probably fine to leave it like this.